ci/build_examples.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Script to build all examples in yew/examples 
+#
+# Not building yew-router nor yewtil examples
+
+# src: https://gist.github.com/fbucek/f986da3cc3a9bbbd1573bdcb23fed2e1
+set -e # error -> trap -> exit
+function info() { echo -e "[\033[0;34m $@ \033[0m]"; } # blue: [ info message ]
+function fail() { FAIL="true"; echo -e "[\033[0;31mFAIL\033[0m] $@"; } # red: [FAIL]
+trap 'LASTRES=$?; LAST=$BASH_COMMAND; if [[ LASTRES -ne 0 ]]; then fail "Command: \"$LAST\" exited with exit code: $LASTRES"; elif [ "$FAIL" == "true"  ]; then fail finished with error; else echo -e "[\033[0;32m Finished $@ \033[0m]";fi' EXIT
+SRCDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" # this source dir
+
+cd $SRCDIR/../examples # ensure current dir is this dir
+
+for EXAMPLE in *
+do  
+    if [[ $EXAMPLE ==  static ]] || [[ $EXAMPLE ==  server ]] || [[ $EXAMPLE ==  target ]] || [[ $EXAMPLE ==  *_wp ]]; then
```

Comment:
What happens when using this script to build the `multi_thread` example? I don't see an exception here

Reply:
Now using one build.sh which has exceptions for `multi_thread`